tables/update-tables.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 # Script to convert table definitions created by Sqoop into
 # bucketed, partitioned tables saved as parquet
-TABLES=$(awk -F"," 'NR>1 {print $1}' tables-list.csv)
+
+# Check if table env var exists, if not, use all tables
```

Comment:
Tiny addition so that you can specify table schema files to change in the same way they're specified for extraction.